rail_scripts/rail-preprocess-parquet

```diff
@@ -181,14 +187,15 @@ def preprocess(cfg):
                 print(input_)
                 return
 
-            has_magnitude, has_flux, has_ra, has_dec = parquet_check_column_presence(
+            has_magnitude, has_flux, has_ra, \
+            has_dec, has_objectId = parquet_check_column_presence(
```

Comment:
Inconsistent coding style: please use underscore notation for variable names: `has_object_id`.